.travis.yml

```diff
@@ -1,3 +1,6 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at <http://mozilla.org/MPL/2.0/>.
 language: python
```

Comment:
lets only have a license header for `/src` files, this means we can drop it for config files, `/assets`, and `/bin` files

this is easy to do w/ `git reset HEAD~1 bin` ...